scripts/asr_language_modeling/ngram_lm/create_lexicon_from_arpa.py

```diff
@@ -26,6 +26,7 @@
 import os
```

Comment:
Remind me, is the arpa support for KenLM or do we have niche instances in NeMo that require arpa?

Reply:
ARPA is used as an intermediate step to create a binary KenLM model.